shakedown/dcos/package.py

```diff
@@ -100,7 +108,7 @@ def install_package(
         future = now + timeout_sec
 
         while now < future:
-            if shakedown.get_service(package_name):
+            if shakedown.get_service(service_name):
```

Comment:
Yep -- this was broken when waiting on services deployed with custom service names.